src/main/java/com/dumbdogdiner/stickyapi/common/util/ShortID.java

```diff
@@ -8,12 +8,13 @@
 import java.util.UUID;
 import java.util.zip.CRC32;
 
+import lombok.experimental.UtilityClass;
 import org.apache.commons.lang.Validate;
 import org.jetbrains.annotations.NotNull;
 
+// TODO this is in the apache validation used in another PR
```

Comment:
Please fix this - won't work on Bungeecord!

Reply:
It should work, because apache validation is a dependancy in heads. I'll set about fixing in heads once this pr is merged

Reply:
Okay, but this is an entirely different branch, and I've asked you before to use guava's preconditions rather than apache validation.

Reply:
bump